### PR TITLE
apps/home: added choosable link to text with image block

### LINKED ISSUE #162: #4899 UX/UI Test - Landing Page
**URL:** https://digitalstrategie-dev.liqd.net/admin/pages/3/edit/preview/
**important screensize:** *Both*
**Comment/Question:** *There is a CTA missing here, the idea of the Landing Page is also to promote other Overview pages meanwhile talking about the project* 

Design on Zeplin: 
Screenshot 2021-10-28 at 12 45 35

Current Status: 
Screenshot 2021-10-28 at 12 42 25

Thanks!

### PR DESCRIPTION
**Pull request by sabinammm**
This closes #162.

@phillimorland I attempted to change the color of the link from blue to black using $link-dark for color in the class that I'm using. But this broke the page as an undefined variable, though I thought this was bootstrap 5 (??). It now looks like this:
image
Should we leave it for now?

### CODE REVIEW

**Comment by phillimorland** [comment]
@sabinammm you can see a list of common variables in `variables.scss` in `assets` folder or to check the entire list, view it in the `node_modules` bootstrap folder or in the docs online, when you get an unknown variable error message, try another known variable to see if it at least works to change the thing you are trying to change even if it's not correct colour ect.  Then check elsewhere for correct one, if still not created and you think it will be used again (colours should always be variables for example, other things you can decide if it's needed). You can add a new variable to `core-variables` but in this case I think you can just use $black right?!

**Comment by sabinammm** [comment]
> @sabinammm you can see a list of common variables in `variables.scss` in `assets` folder or to check the entire list, view it in the `node_modules` bootstrap folder or in the docs online, when you get an unknown variable error message, try another known variable to see if it at least works to change the thing you are trying to change even if it's not correct colour ect. Then check elsewhere for correct one, if still not created and you think it will be used again (colours should always be variables for example, other things you can decide if it's needed). You can add a new variable to `core-variables`

Thank you for the explanation:) $black worked and I'll remember this note for future styling.

File: apps/home/templates/apps_home/blocks/text_with_image_block.html [hunk 1]
```diff
@@ -13,6 +13,11 @@ <h2>
                         {{ value.title }}
                     </h2>
                     {{ value.body|richtext }}
+                    <div>
+                        {% if value.link %}
+                        <a class="quote-ds-block__link" href="{{ value.link.url }}">{% trans 'Read more' %}&#8594;</a>
```
Comment (fuzzylogic2000) [inline apps/home/templates/apps_home/blocks/text_with_image_block.html:18]: The `&#8594;` is the arrow, right? We do normally use font awesome there instead. Just use arrow-right: https://fontawesome.com/v5.15/icons/arrow-right?style=solid But don't forget to add aria-hidden=trrue.

**Review by fuzzylogic2000 — COMMENTED** [review]
Nice! HJust one very tiny remark!

File: apps/home/templates/apps_home/blocks/text_with_image_block.html (hunk 1 above)
Comment (sabinammm) [reply apps/home/templates/apps_home/blocks/text_with_image_block.html]: Awesome, thanks:) I changed it, but I used the same for quote_block so should we change that or just keep in mind that we change it next time we touch it?

Comment (fuzzylogic2000) [reply apps/home/templates/apps_home/blocks/text_with_image_block.html]: Yes, please change it in both and add `aria-hidden="true"`.
(You always need to add either the `aria-hidden` or an `aria-label` to the font awesome images to make then accessible.)

Comment (sabinammm) [reply apps/home/templates/apps_home/blocks/text_with_image_block.html]: Thanks! I added it to the other font awesome images as well, and I hope it's all fine now :)

File: apps/home/templates/apps_home/blocks/quote_block.html [hunk 2]
```diff
@@ -7,15 +7,15 @@
                 {{ self.text }}
                     <div>
                         {% if value.link %}
-                        <a class="quote-ds-block__link" href="{{ value.link.url }}">{% trans 'Read more' %}&#8594;</a>
+                        <a class="quote-ds-block__link" href="{{ value.link.url }}">{% trans 'Read more ' %}<i class="fas fa-arrow-right" aria-hidden="true"></i></a>
```
Comment (fuzzylogic2000) [inline apps/home/templates/apps_home/blocks/quote_block.html:10]: One last thing: There shouldn't be a blank at the end of the translation. Did you add that for the space between the "Read more" and the arrow? Then try to add it between the trans tag and the `<i...`

Comment (sabinammm) [reply apps/home/templates/apps_home/blocks/quote_block.html]: Ok, will remember this. thanks!

**Comment by khamui** [comment]
looking good!

**Pull request merged**